packages/favorites-dialog/src/Favorites.js

```diff
@@ -14,9 +14,24 @@ import EnhancedTable from './EnhancedTable';
 import { fetchData } from './actions';
 
 class Favorites extends Component {
-    componentWillReceiveProps(nextProps) {
+    componentDidMount() {
         // fetch data only the first time the dialog is open
```

Comment:
Seems like an unnecessary comment.

Reply:
True, it does not apply anymore.
Fixed.